### ISSUE TITLE
Feature request - Voyages recommendations to optionally exclude Cadet Challenge crew

### ISSUE THREAD

**Issue by logforgit** [IAmPicard/StarTrekTimelinesSpreadsheet#217]
Problem:
The Voyages recommendations give me crew that I need to be home for the current Cadet Challenge.

Potential Solution:
An option to exclude all the crew eligible for the current Cadet Challenge.

Alternatives:
Right now, I manually type in the Cadet Challenge crew I want to leave home each time.

**Comment by IAmPicard** [comment]
Cadet challenge are 2 and 3 star crew? Hmm... I guess this could come in handy for very early game scenarios; as soon as you get a bunch of 4+ stars, you'll leave all 2-3 stars behind anyway.

**Comment by logforgit** [comment]
Yeah, sorry, I'm only a a month or two into the game. I have several 2 and 3 stars immortalized... still working on 4+. It always wants to take the high level 2's and 3's on the voyage.
